### ISSUE TITLE
Error on dependencies installation

### ISSUE THREAD

**Issue by CaptainJojo** [apollographql/apollo-tooling#2221]
I have an error since today with the global installation 

```
npm install -g apollo
```

My error is

```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for apollo-codegen-core@^0.39.2.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of 'apollo'
npm ERR! notarget 
```

**Comment by nickreynke** [comment]
Got the same. A quick fix for the time being is to install version `2.23.1`:

```
npm install -g apollo@2.23.1
```

(If you are trying to install the Vue CLI: `npm install -g @vue/cli apollo@2.32.1`)

**Comment by abernix** [comment]
Is this still happening for you?

This was definitely the case for a little while after I published; as soon as I caught it I started another release to correct it but I didn't notice this issue until now. 

(An error happened while publishing the packages due to a mistakenly executed process while releasing. See https://github.com/apollographql/apollo-tooling/commit/d03b292bda3d89a87dc67b4f7743706c73689dab)
